packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -2,6 +2,22 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../utils/index';
 
+@mixin iui-tabs-wrapper {
+  display: grid;
```

Comment:
There are some random image changes in pill tabs. Might be because of `grid` but can you verify?

Reply:
Those random changes seem to be happening because of the spacing methods I used in `grid-template-columns`. I'm not sure why it caused some spacing issues in storybook, because it worked correctly in CSS.

Changing the spacing methods in `grid-template-columns` in [`9cc7d64` (#860)](https://github.com/iTwin/iTwinUI/pull/860/commits/9cc7d6498cd07bc4b2aef1316ffbfd72fb1d6abc) got rid of the storybook `Tabs -> Pills` test image change, reduced the number of CSS test image changes, and also made the CSS test image changes cleaner (fewer changes in each image).